SharedPackages/BrowserServicesKit/Sources/Subscription/Managers/V1toV2Bridging/SubscriptionAuthV1toV2Bridge.swift

```diff
@@ -30,7 +30,7 @@ public protocol SubscriptionAuthV1toV2Bridge: SubscriptionTokenProvider, Subscri
     func isFeatureAvailableAndEnabled(feature: Entitlement.ProductName, cachePolicy: APICachePolicy) async throws -> Bool
     /// If the user is allowed to use the feature, base on the TokenContainer entitlements
     /// This is used by VPN and PIR
-    func isFeatureEnabledForUser(feature: Entitlement.ProductName) async -> Bool
+    func isFeatureEnabledForUser(feature: Entitlement.ProductName) async throws -> Bool
```

Comment:
`true` means there are entitlements, `false` means no entitlements, and error is for recoverable errors such as network failures.  We should **NEVER** conflate recoverable errors with regular results.